### DISCUSSION TITLE
Project builds perfectly — added pages appear locally, but not on Vercel

### DISCUSSION THREAD

**Discussion by JohnUW42** [shuding/nextra#1696] [Q&A] [answered]
Hi!

I am asking for you help with the following, please.

My project builds successfully both locally and on Vercel.

However I cannot see a new page added to a built project on Vercel.

As a base I used Vercel's "Nextra: Docs Starter Kit" template.

I can make changes in Nextra config (like footer text changes) and those changes are reflected when the project is built on Vercel. But a new pages does not simply appear on Vercel — although the same page does appear on exactly the same build locally.

I cannot quite understand this.
And running a new browser session in private/incognito mode (just in case of cache problem) does not help.
Also, the name of the page not appearing on Vercel is being shown during the build process on Vercel.

Please help!

Thank you!

**Comment by dimaMachina** [comment, marked as answer]
https://github.com/shuding/nextra/issues/1564#issuecomment-1447419595

**Reply by JohnUW42** [reply]
Thank you!

Redeploying the same source code, which clears cache on Vercel in the building process, fixes the problem!
